rel/files/mongooseim.cfg

```diff
@@ -427,126 +423,66 @@
-{{{rdbms_server_type}}}
-
-%%% ====================
-%%% RIAK SETUP
-%%% ====================
-
-{{{riak_server}}}
+%% == ODBC (MSSQL) ==
+%%  {rdbms, global, default, [{workers, 10}],
+%%   [{server, "DSN=mongooseim;UID=mongooseim;PWD=mongooseim"}]},
 
-%%% ==================================
-%%% POOLS OF OUTGOING HTTP CONNECTIONS
-%%% ==================================
-%%
-%%{http_connections, [{conn1, [{server, "http://server:8080"},
-%%                              {pool_size, 50}]}
-%%                    ]}.
-%%
+%% == Elastic Search ==
+%%  {elastic, global, default, [], [{host, "elastic.host.com"}, {port, 9042}]},
 
-%%% ====================
-%%% CASSANDRA SETUP
-%%% ====================
+%% == Riak ==
+%%  {riak, global, default, [{workers, 20}], [{address, "127.0.0.1"}, {port, 8087}]},
 
-{{{cassandra_servers}}}
+%% == HTTP ==
+%%  {http, global, conn1, [{workers, 50}], [{server, "http://server:8080"}]},
 
-%% Connect to localhost:9042, one worker, one pool, shared between all hosts
-%%{cassandra_servers, [{default, []}]}.
+%% == Cassandra ==
+%%  {cassandra, global, default, [{workers, 100}],
+%%    [
+%%      {servers, [
+%%                 {"cassandra_server1.example.com", 9042},
+%%                 {"cassandra_server2.example.com", 9042},
+%%                 {"cassandra_server3.example.com", 9042},
+%%                 {"cassandra_server4.example.com", 9042}
+%%                ]},
+%%      {keyspace, "big_mongooseim"}
+%%    ]}
 
-%% ALTERNATIVE CONFIGURATION: Custom server address
-%% - 100 connections to all four Cassandra servers in total
-%% - Keyspace "big_mongooseim"
+%% == Extra options ==
 %%
-%%{cassandra_servers,
-%% [
-%%  {default, 100,
-%%   [
-%%    {servers,
-%%     [
-%%      {"cassandra_server1.example.com", 9042},
-%%      {"cassandra_server2.example.com", 9042},
-%%      {"cassandra_server3.example.com", 9042},
-%%      {"cassandra_server4.example.com", 9042}
-%%     ]
-%%    },
-%%    {keyspace, "big_mongooseim"}
-%%   ]
-%%  }
-%% ]
-%%}.
-
-%% ALTERNATIVE CONFIGURATION: Two pools: for mam and for mam_muc
-%% - Each pool uses different keyspace
-%% - Module mod_mam_cassandra_arch uses pool "mam" and keyspace "mam_space"
-%% - Module mod_mam_muc_cassandra_arch uses pool "mam_muc" and keyspace "mam_muc_space"
+%% If you use PostgreSQL, have a large database, and need a
+%% faster but inexact replacement for "select count(*) from users"
 %%
-%% Set pool_name for modules like this (not just uncomment this example!)
-%% {modules,
-%%    [{mod_mam_cassandra_arch, [{pool_name, mam}]},
-%%     {mod_mam_muc_cassandra_arch, [{pool_name, mam_muc}]}]}.
+%%{pgsql_users_number_estimate, true}.
 %%
-%%{cassandra_servers, [{mam, [{keyspace, "mam_space"}]}, {mam_muc, [{keyspace, "mam_muc_space"}]}]}.
-
-%%% ====================
-%%% ELASTICSEARCH SETUP
-%%% ====================
-
-{{{elasticsearch_server}}}
+%% rdbms_server_type specifies what database is used over the RDBMS layer
+%% Can take values rdbms, pgsql, mysql
```

Comment:
`rdbms` should be `mssql` here. I don't know why it was `odbc` before

Reply:
I think in this place it meant "true" ODBC, not quasi-ejabberd-odbc.